operator/pkg/ciliumendpointbatch/endpointbatch.go

```diff
@@ -283,6 +284,12 @@ func (c *CiliumEndpointBatchController) syncCeb(key string) error {
 			return err
 		}
 	}
+	// Update metrics
```

Comment:
Are any of the error case above measured automatically, or there is no need in tracking those?

Reply:
Good point, i'll add metrics for the erros.